tests/lcb/compiler/frontend/variadic.compilertest

```diff
@@ -17,7 +17,7 @@
 %% Variadic parameter must be the last in the parameter list
 %TEST VariadicLastInForeignHandler
 module compiler_test
-foreign handler TestVariadic(in pA as any, ...) returns nothing binds to ""
+foreign handler TestVariadic(in pA as any, ...) returns nothing binds to "dummy"
```

Comment:
@runrevmark this is now required because we now get parser errors on binding strings... aka `MCScriptThrowMissingFunctionInForeignBindingError`